samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_ca.properties

```diff
@@ -360,3 +360,7 @@ reader_lt=i menor que
 reader_end=punts.
 reader_earned=obtinguts
 reader_starting=comen\u00e7ant
+
+# S2U-15
+submission_nav_next_submission=Seg\u00FCent lliurament
+submission_nav_previous_submission=Lliurament anterio
```

Comment:
Anterior lliurament

Reply:
Lliurament seg\u00fcent?

Reply:
Let's be consistent with ES, 

Seg\u00FCent lliurament
Anterior lliurament

Makes sense?

Reply:
To be consistent with ES, it would be:
```
Lliurament anterior
Lliurament seg\u00fcent
```
like:
```
Env\u00edo anterior
Env\u00edo siguiente
```
no?

Reply:
YYYYYYYY